### ISSUE TITLE
[TW-591] Autopush after merge fails because 'ssh://' gets cut off

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#626]
_Johannes Schlatow on 2011-03-10T19:30:37Z says:_ 

 From IRC:
<pre>
14:23:51 < hrubi> hi, I got "merge.default.uri=ssh://kikina-task/.task/" in my 
                  .taskrc (kikina-task is defined in .ssh/config). If I do merge, 
                  it downloads data fine, but it tries to push back to 
                  'kikina-task/.task/' and it whines that it's not a local 
                  directory (it should try to push over ssh instead). I'm going to 
                  elaborate on different format of the merge uri.
14:24:25 < hrubi> anyway it's strange that it downloads fine, but the push fails
14:25:20 < hrubi> it works with configuration 
                  "merge.default.uri=kikina-task:.task/"
</pre>

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-03-10T19:30:37Z
Modified: 2014-02-09T02:06:29Z
```

**Comment by taskwarrior** [comment]
_Johannes Schlatow on 2011-03-10T19:33:37Z says:_ 

 (at)hrubi:

You can try the attached patch or compile 1.9.5 from git repository.

**Comment by taskwarrior** [comment]
_Jan Hruban on 2011-03-11T10:25:24Z says:_ 

 I confirm that the patch fixes the problem with version 1.9.4.

**Comment by pbeckingham** [comment]
[TW-591_0001-Merge-Uri-support.patch.txt](https://github.com/GothenburgBitFactory/taskwarrior/files/1756180/TW-591_0001-Merge-Uri-support.patch.txt)
